tic-tac-toe/src/app/screens/Login/fields/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const customInput = props => {
```

Comment:
We use function to create component. Not forget to name component file in Upper Case.